src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle() {
+        super();
+        firstLeg = new Random().nextInt(100);
+        secondLeg = new Random().nextInt(100);
```

Comment:
Random value generation should be handled in the `FigureSupplier` class, as per the checklist: 'You can pass random values to the constructor of a figure instead of generating them inside figure classes.'